src/lib/esnext.string.d.ts

```diff
@@ -0,0 +1,7 @@
+interface String {

```

Comment:
Just in case there are any existing `--lib esnext.string` options in the wild, I would recommend `esnext.string` reference `es2021.string`.

```suggestion
/// <reference lib="es2021.string" />
interface String {
```